package/firmware/ipq-wifi/Makefile

```diff
@@ -17,7 +17,7 @@ endef
 # Please send a mail with your device-specific board files upstream.
```

Comment:
Have you sent a mail with your board data to the ath10k@lists.infradead.org yet?
I've look in the ath10k ML archive, but I didn't find anything related to the EA6350v3 yet?

Please send a mail with your device-specific board files upstream.
You can find instructions and examples on the linux-wireless wiki:
<https://wireless.wiki.kernel.org/en/users/drivers/ath10k/boardfiles>

Reply:
http://lists.infradead.org/pipermail/ath10k/2019-January/012662.html

Thank you!

Reply:
Question: once the file is sent... what's next? Shall I delete the custom `generate-ipq-wifi-package`. If that's true then: where do I found the board data?
What should I do now?

Reply:
> Shall I delete the custom generate-ipq-wifi-package. If that's true then: where do I found the board data?

Keep the file as is for now. The custom boardfile will get deleted once the official linux-firmware.git supports the board. But this is can take maybe like a year or so.

> What should I do now?

The board-data (well the EU version of it, but that's fine) was sent to the ath10k-devel.
So as far as the boarddata for the Linksys EA6350v3 is concerned: it's all fine. Once
the board-data moved through [ath10k-firmware](https://github.com/kvalo/ath10k-firmware) and into linux-firmware.git the custom board files can be deleted. But for now, it's fine.  :+1: